### DISCUSSION TITLE
Cannot convert from Windows.UI.Xaml.DragEventArgs to Android.View.DragEventArgs

### DISCUSSION THREAD

**Discussion by jpageacsys** [unoplatform/uno#12019] [Q&A]
I've just been commenting this out for now to get the app to work, but one thing I was not able to resolve when moving my app from UWP to android was all functions with DragEventArgs attached to TabViews show Cannot convert from Windows.UI.Xaml.DragEventArgs to Android.View.DragEventArgs

For example here's my function:
`private void TabView_Drop(object sender, DragEventArgs e)
        {
            TabHelper.Drop(TabView);
        }`
Here's my XAML
`<muxc:TabView 
                Drop="TabView_Drop"
                CanDragTabs="True"
                AllowDrop="True"`

The autogenerated code from the page shows this:
```
#if __ANDROID__
using _View = Android.Views.View;
#elif __IOS__
using _View = UIKit.UIView;
#elif __MACOS__
using _View = AppKit.NSView;
#elif UNO_REFERENCE_API || NET461
using _View = Windows.UI.Xaml.UIElement;
#endif
```
I'm guessing I have to change the argument to match, but I'm not sure how to do this.

**Comment by jeromelaban** [comment]
Thanks for the report. You may need to explicitly specify the type of `Windows.UI.Xaml.DragEventArgs` in your `TabView_Drop` method, as it's likely to be an inner type of `Android.View` and the C# compiler picks it first.
